int8 cell
int8 mobileRobot
int8 red
int8 blue
int8 yellow
int8 status 
